BEP67.md

```diff
@@ -0,0 +1,48 @@
+# BEP-67: Differential Order Expiration
+
+- [BEP-67: Differential Order Expiration](#bep-67-differential-order-expiration)
+  - [1. Summary](#1-summary)
+  - [2. Abstract](#2-abstract)
+  - [3. Status](#3-status)
+  - [4. Motivation](#4-motivation)
+  - [5. Specification](#5-specification)
+    - [5.1 Order Expiration](#51-order-expiration)
+    - [5.2 Change Impact](#52-change-impact)
+  - [6. License](#6-license)
+
+## 1. Summary 
+
+This BEP describes an enhancement of the [Order Expiration](https://docs.binance.org/faq.html#what-is-order-expire).  
+
+## 2. Abstract
+
+Currently orders on Binance Chain will be expired after 72 hours. This may lead to high frequency of order placement, which is not very convenient to users. 
```

Comment:
not exactly 72 hours. "high frequency" is a word often used in "high frequency trading", which means trade multiple  time in a sub-second time period. I suggest we say:

"Order cannot live long on the market even their price stays competitive, which is not convenient and incur cost to traders."